notte/env.py

```diff
@@ -147,7 +147,7 @@ async def chat(self, text: str) -> str:
             return self._parser.textify(obs)
         elif endpoint == "step":
             step_params = self._parser.step(text)
-            obs = await self.step(step_params.action_id, step_params.params, enter=False)
+            obs = await self.step(step_params.action_id, step_params.params, enter=True)
```

Comment:
I don't think that this is needed

Reply:
For me needed otherwise it doesn't succeed to insert something (like confirm to put the departure city)

Reply:
I think it's none by default ```
async def step(
        self,
        action_id: str,
        params: dict[str, str] | str | None = None,
        enter: bool | None = None,
    )
   ```